docs/source/markdown/podman.1.md

```diff
@@ -44,7 +44,11 @@ Path of the conmon binary (Default path is configured in `containers.conf`)
 
 #### **--events-backend**=*type*
 
-Backend to use for storing events. Allowed values are **file**, **journald**, and **none**.
+Backend to use for storing events. Allowed values are **file**, **journald**, and
+**none**. When *file* is specified, the events are stored under the
+**$XDG_RUNTIME_DIR/libpod** tree. The exact sub-directory/location may vary
```

Comment:
This is not correct for Podman as root - we default to `/var/run/libpod` in that case.

Reply:
A better way to word this would be "events are stored under the Podman temporary files directory (`tmpdir`)

Reply:
okay, thanks.  My concern then is, if someone is specifying `file`, there's a good chance they will (eventually) be concerned about where the file is (or will be).  Especially since the definition changes for root vs rootless.

Perhaps, is "The podman temporary directory" defined someplace else in the docs?  If so, I would feel more comfortable simply referring to it as you suggest.

Reply:
...wait, is this the same `tmpdir` as used/described for the `--tmpdir` option?